src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-details/rgw-multisite-details.component.ts

```diff
@@ -546,20 +546,20 @@ export class RgwMultisiteDetailsComponent implements OnDestroy, OnInit {
 
   delete(node: TreeNode) {
     if (node.data.type === 'realm') {
-      this.modalRef = this.modalService.show(CriticalConfirmationModalComponent, {
+      const modalRef = this.cdsModalService.show(CriticalConfirmationModalComponent, {
```

Comment:
why this change ? how is it fixing the issue ?

Reply:
Here we were using bootstrap modal service instead of carbon modal service. 
Solution -> Replacing Bootstrap Modal Service (**ModalService** with **ModalCdsService**